### ISSUE TITLE
Ghost: Import Marea Proxies

### ISSUE THREAD

**Issue by seandenigris** [seandenigris/SeansOutbox#3]
From http://forum.world.st/Proxies-Subclassing-nil-tp4816443p4816618.html:
something that would be super super cool is to take the proxies I implemented for Marea and adapt them as custom Ghost proxies. Because the idea of Ghost was mostly the design, not the code itself. The code was very very little.  So in Marea proxies you can see how I:

1) Created a few different proxies for different purposes. For example, I created special proxies for classes and metaclasses. And each of them implements custom messages to avoid unnecessary interception (for example, MARMetaclassProxy implements #isMeta answering true. Both, proxy for Class and Metaclass implement:

```
isKindOf: aClass
    ^ (aClass == Class) 
        ifTrue: [ true ]
        ifFalse: [ super isKindOf: aClass  ] "would be intercepted"
```

and so on....

So basically....you are able to implement custom proxies and implement custom messages so that you avoid intercepting them. 

2) In Marea, I have multiple proxies that store the information differently. So for example, Marea proxies needed a graph id and a "position" (an integer). If both were small enough, I was doing a bitShift and storing them in a super compact proxy class that would have only one small integer. 

So....what I want to say is that while ghost code is OK, the most interesting usage and development of those proxies is in Marea, in the category 'Marea-Proxies'.
